src/pages/home/report/ReportActionItem.js

```diff
@@ -56,7 +63,8 @@ class ReportActionItem extends Component {
     shouldComponentUpdate(nextProps, nextState) {
         return this.state.isPopoverVisible !== nextState.isPopoverVisible
             || this.props.displayAsGroup !== nextProps.displayAsGroup
-            || !_.isEqual(this.props.action, nextProps.action);
+            || !_.isEqual(this.props.action, nextProps.action)
+            || this.props.displayNewIndicator !== nextProps.displayNewIndicator;
```

Comment:
NAB, probably better to put this check before the deep equality check before it so that we avoid a more expensive calculation in favor of a cheaper one.